### PR TITLE
Completed Week 3, Step 6: Extras

### PR DESCRIPTION
**Pull request by danieldortega77**
Add buttons to change maximum number of visible comments and delete all comments.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -22,8 +22,19 @@ async function getComments() {
...
+  }
+
+  if (commentsElement.innerHTML === '') {
+    document.getElementById("comment-section").style.visibility="hidden";
+  } else {
+    document.getElementById("comment-section").style.visibility="visible";
+=======
   for (const comment of comments) {
     commentsElement.appendChild(createCommentElement(comment))
+>>>>>>> master
```
Comment (dslucas) [inline portfolio/src/main/webapp/script.js]: Seems like there's some merge artifacts?

Comment (danieldortega77) [reply portfolio/src/main/webapp/script.js]: Oops, I didn't realize this happened. Done.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -19,11 +19,17 @@ async function getComments() {
...
   commentsElement.innerHTML = '';
 
-  for (const comment of comments) {
-    commentsElement.appendChild(createCommentElement(comment))
+  for (var comment of comments) {
+    commentsElement.appendChild(createCommentElement(comment));
+  }
+
+  if (commentsElement.innerHTML === '') {
+    document.getElementById("comment-section").style.visibility="hidden";
```
Comment (dborkan) [inline portfolio/src/main/webapp/script.js]: nit: can you change " to ' and also add spaces around the `=`?

**Review by dslucas — APPROVED** [review]

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (danieldortega77) [reply portfolio/src/main/webapp/script.js]: Will do!

**Pull request merged**